api_app/analyzers_manager/classes.py

```diff
@@ -1,6 +1,7 @@
 # This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
 # See the file 'LICENSE' for copying permission.
 
+from datetime import datetime
```

Comment:
`from django.utils import timezone` and then use `timezone.now()`. Otherwise django throws naive datetime warnings.